### ISSUE TITLE
Loinc 32799-9:  "Glycosaminoglycans/Creatinine [Mass Ratio] in Urine"

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#3603]
Suggest creating a new term for Loinc: 32799-9 [Glycosaminoglycans/Creatinine [Mass Ratio] in Urine] 
New term label:Glycosaminoglycans/Creatinine in Urine
New term comment (if any):
Your biocurator ID for loinc2hpo (if desired): OHSU:JP

**Comment by jpgourdine** [comment]
The same term goes for these LOINC ID <img width="996" alt="screen shot 2018-05-09 at 2 33 36 pm" src="https://user-images.githubusercontent.com/16328677/39840937-02fbcef0-5396-11e8-8be3-7afd24298e5e.png">

**Comment by pnrobinson** [comment]
Hi JP -- please note the HP term should be the abnormality that can be detected by the test, and not the measurement or trait (Glycosaminoglycans/Creatinine in Urine). Thus, Increased glycosaminoglycan to creatinine urine ratio might for instance be the label...what is the medical indication for this test?

**Comment by jpgourdine** [comment]
Thank you Peter for the precisions.
The medical indication: glycosaminoglycans (GAG) in the urine are markers for mucopolysaccharidoses (MPS) [look here](https://s.details.loinc.org/LOINC/32799-9.html?sections=Comprehensive)
I guess all of these terms are equivalent to Urinary glycosaminoglycan excretion (HP:0003541)

**Comment by pnrobinson** [comment]
I think this is simply the HPO term
https://hpo.jax.org/app/browse/term/HP:0003541
This is a present/absent call, right @jpgourdine

**Comment by jpgourdine** [comment]
Hi Peter,

Correct, just presence/absence call.

—
Jean-Philippe Gourdine, PhD
Metabolomics & Glycomics Data Analyst
Translational & Integrative Sciences Lab (Tislab<https://tislab.org/>)
Oregon Clinical & Translational Research Institute (OCTRI<http://www.ohsu.edu/xd/research/centers-institutes/octri/>)
Oregon Health & Science University (OHSU<https://www.ohsu.edu/xd/>)
3181 SW Sam Jackson Park Road, Portland, Oregon 97239

**Comment by pnrobinson** [comment]
closing, adding issue on LOINC tracker
